src/core/qgsapplication.cpp

```diff
@@ -577,6 +585,35 @@ void QgsApplication::setThemeName( const QString &themeName )
   ABISYM( mThemeName ) = themeName;
 }
 
+QString QgsApplication::resolvePkgPath()
```

Comment:
Whole point of this method is to just resolve the package path so we can find what we need pre calling init.  This could set the variables but seems odd to do it here when we just need it for the init process.